python/lolopy/learners.py

```diff
@@ -83,6 +85,11 @@ def fit(self, X, y, weights=None):
         # Get the model out
         self.model_ = result.getModel()
 
+        # Store the feature importances
+        feature_importances_java = result.getFeatureImportance().get()
+        feature_importances_bytes = self.gateway.jvm.io.citrine.lolo.util.LoloPyDataLoader.send1DArray(feature_importances_java)
+        self.feature_importances_ = np.frombuffer(feature_importances_bytes, 'float')
```

Comment:
Does sklearn define a getter for the feature importances? If so, could you add a test that accesses it?

Reply:
[sklearn does define a getter](https://github.com/scikit-learn/scikit-learn/blob/7389dbac82d362f296dc2746f10e43ffa1615660/sklearn/ensemble/forest.py#L365) because they only compute importances on-demand. I always store them (to avoid having to store the entire training result class), so don't use a getter pattern. That said, I could implement one so we get "model not fitted" errors just like sklearn.

Reply:
With the code as it is, how is a user supposed to access the feature importances?

Reply:
They are available as the `.feature_importances_` attribute, which mirrors sklearn's RandomForest and is something I have failed to document

Reply:
Weird. (https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html#sklearn.ensemble.RandomForestRegressor.feature_importances_)  :man_shrugging: